src/payment/PaymentPage.jsx

```diff
@@ -25,9 +25,12 @@ class PaymentPage extends React.Component {
 
   renderEmptyMessage() {
     return (
-      <p>
-        {this.props.intl.formatMessage(messages['payment.empty.basket'])}
-      </p>
+      <div className="card">
+        <div className="card-body">
+          <h5 className="card-title">{this.props.intl.formatMessage(messages['payment.empty.basket'])}</h5>
+          <p className="card-text">If you attempted to make a purchase, you have not been charged. Return to your <a href="https://courses.edx.org/dashboard">dashboard</a> to try again, or <a href="https://courses.edx.org/support/contact_us">contact edX E-commerce Support</a>.</p>
```

Comment:
i18n

Reply:
Checkout this example of getting links inside a FormattedMessage: https://github.com/edx/frontend-app-account/blob/master/src/account-settings/delete-account/BeforeProceedingBanner.jsx#L22